### PR TITLE
fix: cleanup of the instance policy output

### PR DESCRIPTION
**Pull request by MatthewLemmond**
### Description

Addresses the odd formatting/data types in the output as shown in https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/issues/417, there is a field from the provider that doesn't appear in the provider reference documentation so it is not present in this output

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by shemau — DISMISSED** [review]
LGTM

File: main.tf [hunk 1]
```diff
@@ -45,6 +45,17 @@ resource "ibm_kms_instance_policies" "key_protect_instance_policies" {
   }
 }
 
+locals {
```
Comment (ocofaigh) [inline main.tf:48]: Is there a provider issue for this workaround? If so please paste it as a code comment here

Comment (MatthewLemmond) [reply main.tf:48]: Just opened https://github.com/IBM-Cloud/terraform-provider-ibm/issues/5163

Comment (MatthewLemmond) [reply main.tf:48]: added a link to the provider issue in the local, creating an issue on our side to track the provider issue/remove this workaround once output issue is fixed

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**